juntagrico/templates/manage_shares.html

```diff
@@ -63,11 +63,13 @@ <h4>{% trans "Übersicht" %}</h4>
                                     {% blocktrans %}Du brauchst mindestens 1 {{ v_share }}{% endblocktrans %}
                                 {% elif forloop.counter <=  required %}
                                     {% blocktrans %}Diese/s/r {{ v_share }} wird in einem/r {{ v_subscription }} benötigt{% endblocktrans %}
-                                {% elif  share.cancelled_date is None and share.paid_date is None %}
+                                {% elif share.cancelled_date is None and share.paid_date is None %}
                                     {% blocktrans %}Diese/s/r {{ v_share }} wurde noch nicht als bezahlt verbucht{% endblocktrans %}
+                                {% elif ibanempty %}
+                                    {% blocktrans %}Kann gekündigt werden, sobald eine IBAN hinterlegt ist{% endblocktrans %}
                                 {% elif  share.cancelled_date is None and share.paid_date is not None %}
                                     <a href="{% url 'share-cancel' share.id %}"class="btn btn-success"
-                                       onclick="return confirm('{% blocktrans %} Möchtest du sicher diese/s/n {{ v_share }} verbindlich kündigen? {% endblocktrans%}')">
+                                       onclick="return confirm('{% blocktrans %} Möchtest du sicher diese/s/n {{ v_share }} verbindlich zum {{ canceldate }} kündigen? {% endblocktrans%}')">
```

Comment:
Using the chance to fix the sentence
```suggestion
                                       onclick="return confirm('{% blocktrans %} Möchtest du diese/s/n {{ v_share }} verbindlich zum {{ canceldate }} kündigen? {% endblocktrans%}')">
```